post_office/template/backends/html_email.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
```

Comment:
Can we rename this file to `template/backends/__init__.py` and the engine to `PostOffice`? This way we can refer to this engine as `'post_office.template.backends.PostOffice`

Reply:
I can rename it to `template/backends/post_office.py` if you want, but not to `__init__.py`. Reason is that this name is used in the `settings.py` for `POST_OFFICE['TEMPLATE_ENGINE']` and therefore we need a meaningful name here.
This btw. was the main reason, to use `html_email` rather than `post_office` as the template's engine name, because I wanted to emphasize that it is intended for HTML email rendering.

Reply:
How about moving this to `/post_office/template/backends.py` then? Name the engine `HTMLEmailEngine` or `HTMLEmail` so it can be specified as `'post_office.template.backends.HTMLEmailEngine`.

The built in Jinja2 engine can be imported as `django.template.backends.jinja2.Jinja2`. I think this is a better naming convention than `DjangoTemplates` (it's not a template).

Reply:
The problem are these functions: `django.template.loader.get_template`, `django.template.loader.select_template` and  `django.template.loader.render_to_string`. They all take a parameter `using`, which in our case would be `get_template('/path/to/message.html', using='html_email')`. Here `html_email` is the name of the module in `backends`. If we move it to `backends.py`, one would have to call `get_template('/path/to/message.html', using='backends')` – not a very good name.

As said, I have no problem using `post_office` here.

Reply:
I just renamed the template engine to `post_office.PostOfficeTemplates`. I would really linke to keep "Templates" inside the name, because that is what Django does with its own template engine: `django.template.backends.django.DjangoTemplates`.